app/dashboard/helpers.py

```diff
@@ -628,15 +659,14 @@ def merge_bounty(latest_old_bounty, new_bounty, metadata, bounty_details, verbos
         new_bounty.canceled_on = canceled_on
         new_bounty.save()
 
-    # migrate fulfillments, and only take the ones from
-    # fulfillments metadata will be empty when bounty is first created
     fulfillments = bounty_details.get('fulfillments', {})
     if fulfillments:
         handle_bounty_fulfillments(fulfillments, new_bounty, latest_old_bounty)
         url = normalize_url(new_bounty.github_url)
         for inactive in Bounty.objects.filter(
             current_bounty=False, github_url=url
         ).nocache().order_by('-created_on'):
+            # TODO: evalute if this can be removed
```

Comment:
As far as I can tell, this filter should always return nothing, right?

Reply:
Yes that is correct ! It's just a fallback code

Reply:
Meh lets just leave it I guess